tools/pyboard.py

```diff
@@ -582,12 +583,18 @@ def fname_remote(src):
         return src
 
     def fname_cp_dest(src, dest):
-        src = src.rsplit("/", 1)[-1]
+        if separator == "/":
```

Comment:
This if/else can just be `src = src.rsplit(os.path.sep, 1)[-1]`

But that should actually can be simplified to just `_, src = os.path.split(src)`

Reply:
`_, src = os.path.split(src)` ，great！